### PR TITLE
docker-up: drop the `kernel.domainname` as rootless container currently fail

### PR DESCRIPTION
**Pull request by utam0k**
## Description
Drop the `kernel.domainname` as rootless container currently fail. If this does not work, consider using oci hooks.
- https://github.com/opencontainers/runc/issues/2091
- https://github.com/opencontainers/runtime-spec/issues/592

## Related Issue(s)

Relate #7440
This will work, but I'm not sure if it will work as expected by the issue reporter, so I'll leave it out of the fix.

## How to test

Check that the following commands work on gitpod
```console
$ cd components/docker-up/runc-facade/
$ go build
$ sudo cp runc-facade /usr/bin/runc-facade
$ docker run --domanname "example.com" osixia/openldap:stable
```

## Release Notes

```release-note
None
```

## Documentation

No

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **utam0k** after the PR has been reviewed.

Associated issue: *#7440*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/docker-up/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/docker-up/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/docker-up/runc-facade/main.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func main() {
 	}
 
 	if useFacade {
-		err = createAndRunc(runcPath)
+		err = createAndRunc(runcPath, log)
```
Comment (csweichel) [inline components/docker-up/runc-facade/main.go:41]: no need to pass in `log` here ... you can just use the functions from `common-go/log` directly.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 96a320c76697c51a28ea7dadcb6be2158c30891c</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7705#issuecomment-1017482348" title="LGTM">csweichel</a>*

Associated issue: *#7440*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/docker-up/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/docker-up/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**